### PR TITLE
Add 36 Cucumber test for account and product2

### PR DESCRIPTION
**Pull request by edsonarios**

### CODE REVIEW

**Review by JorgeRCV — APPROVED** [review]

File: src/test/resources/schemas/response.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      "type": "array",
+      "items": {}
+    }
+  },
+  "required": [
+    "id",
+    "success",
+    "errors"
+  ]
+}
```
Comment (Paola-Xaq) [inline src/test/resources/schemas/response.json:21]: missing black space at the end

File: src/test/resources/features/product2.feature [hunk 2]
```diff
@@ -0,0 +1,130 @@
+Feature: Request for product2 feature
+  All request for the endpoint case
+
+  @ShowAllProduct2
+  Scenario: Get all Product2
+    Given I build "GET" request
+    When I execute "PRODUCT2" request
+    Then Status response of request should be "OK"
+
+  @CreateDeleteProduct2
```
Comment (Paola-Xaq) [inline src/test/resources/features/product2.feature:10]: maybe the tags should be more descriptive

File: src/test/resources/features/account.feature [hunk 3]
```diff
@@ -32,3 +32,98 @@ Feature: Request for account feature
...
+  Scenario: Response body is the same name for A Account Created
+    Given I build "GET" request
+    When I execute for "ACCOUNT_ID" request with param
+    Then The account response body name of the attribute is the same as the wait and request must be "OK"
+
+  @CreateDeleteAccount
+  Scenario: Attribute response body is kind account for A Account Created
+    Given I build "GET" request
+    When I execute for "ACCOUNT_ID" request with param
+    Then The account response body kind of the attribute is the same as the wait and request must be "OK"
```
Comment (Paola-Xaq) [inline src/test/resources/features/account.feature:129]: missing final line

**Review by Paola-Xaq — APPROVED** [review]
There are some things to fix, do it in the next PR

**Pull request merged**